### DISCUSSION TITLE
stantargets micro-case-study

### DISCUSSION THREAD

**Discussion by mike-lawrence** [wlandau/stantargets#32] [Show and tell]
I used stantargets over the weekend to check the math/compute of a trick I came up with, and [posted the results here](https://discourse.mc-stan.org/t/psa-using-stantargets-for-sbc-esque-checks-big-speedups-for-big-gaussian-likelihoods/21425) :) 

As my other posts for help/discussion here during the process indicate, I had a few points of learning to get things working, but came away feeling that the approach stantargets takes is sound (consistent with my prior ;) ).

**Comment by wlandau** [comment]
Awesome, thanks for sharing!
